README.md

```diff
@@ -38,4 +38,6 @@ To generate your own secure mTLS certificates, check
 [this tutorial](https://medium.com/weekly-webtips/how-to-generate-keys-for-mutual-tls-authentication-a90f53bcec64).
 
 ## Planned work
-- Add a destination for gRPC server. 
\ No newline at end of file
+- Add a destination for gRPC server. 
+
+![scarf pixel](https://static.scarf.sh/a.png?x-pxid=)
```

Comment:
Looks like it's incomplete?

Reply:
Sorry! would you please share a scarf pixel for the same?

Reply:
Yup, here it is! https://static.scarf.sh/a.png?x-pxid=fdabe747-b944-4ab3-a349-b8d3e2f2e19c

Reply:
Added. Thanks!